README.md

```diff
@@ -23,9 +23,11 @@ Check out the [examples notebook](examples/swifter_apply_examples.ipynb), along
 ```
 $ pip install -U pandas # upgrade pandas
 $ pip install swifter # first time installation
+$ pip install swifter[progress_bar] # first time installation including dependency for rich progress bar in jupyter notebooks
 $ pip install swifter[groupby] # first time installation including dependency for groupby.apply functionality
 
 $ pip install -U swifter # upgrade to latest version if already installed
+$ pip install -U swifter[progress_bar] # upgrade to latest version to include dependency for rich progress bar in jupyter notebooks
```

Comment:
I'd suggest `swifter[notebook]` or `swifter[jupyter]` since we do not need `ipywidgets` for non-notebook environments.

Reply:
yeah i agree, i think ill do `notebook` instead

Reply:
updated https://github.com/jmcarpenter2/swifter/pull/220/commits/8ef2357ecdcc07e243a4b5a57bf17bf7b0b7c84b